catalog/validator/catalog-validator-metacardduplication/src/main/java/org/codice/ddf/validator/metacard/duplication/DuplicationValidator.java

```diff
@@ -227,9 +227,15 @@ private ValidationViolation reportDuplicates(final Metacard metacard, String[] a
         return filters;
     }
 
-    private SourceResponse query(Set<Attribute> attributes) {
+    private SourceResponse query(Set<Attribute> attributes, String originalId) {
 
-        final Filter filter = filterBuilder.anyOf(buildFilters(attributes));
+        final Filter filter = filterBuilder.allOf(filterBuilder.anyOf(buildFilters(attributes)),
+                filterBuilder.not(filterBuilder.attribute(Metacard.ID)
```

Comment:
It doesn't look like GeoTools supports notEqualTo for some reason.  In this case the `not` filter should be fine.